include/hc.hpp

```diff
@@ -316,6 +318,33 @@ class accelerator_view {
      */
     completion_future copy_async(const void *src, void *dst, size_t size_bytes);
 
+
+    /**
+     * Copies size_bytes bytes from src to dst.  
+     * Src and dst must not overlap.  
+     * Note the src is the first parameter and dst is second, following C++ convention.  
+     * This is an asynchronous copy command, and this call may return before the copy operation completes.
+     *
+     * The copy command will be implicitly ordered with respect to commands previously enqueued to this accelerator_view:
```

Comment:
This API takes a pointer of an hc::accelerator rather than a pointer of an hc::accelerator_view so how would the copy command or the user know which accelerator_view the copy command is synchronized with?  Unless the copy command is synchronized with all accelerator_views associated with that specific accelerator?